pkg/ocm/client.go

```diff
@@ -121,6 +121,15 @@ func (c *Client) newConnection() error {
 	return nil
 }
 
+type Scope string
```

Comment:
Scope -> RoleType

Suggest to add the "type" fo avoid confusing with "Role" property in line 142 below

Reply:
This is how I originally started but I changed role to scope so it won't be confusing when comparing "role" to "scope" in authorization context.